cmd/build/helmify/main.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+)
+
+var (
+	outputDir  = flag.String("output-dir", "manifest_staging/helm/gatekeeper", "The root directory in which to write the Helm chart")
+	useCRDsDir = flag.Bool("use-crds-dir", false, `Use the "crds" subdirectory, which requires Helm v3`)
```

Comment:
do we still want to keep Helm 2 as default or move to Helm 3?

Reply:
No clue. I suspect there are more helm 2 users out there currently, and helm-2-style charts are compatible with helm 3, but the reverse is not true, which leads me to guess helm 2.

I'm open to changing this if we want, but maybe that could be a follow-on PR to reduce the delta?

Reply:
sgtm. We can investigate supporting Helm 3 as a follow-up (hopefully we can make it forwards compat).

Reply:
Is the existing chart not forward compatible? The only difference I'm aware of is CRD handling and maybe some changes to the library the template uses?

Reply:
I don't believe helm 3 client installs CRDs when we use with helm 2 chart, because of the missing `crd-install` hook

Reply:
Ugh, that seems like something helm should address. Breaking backwards compatibility makes ecosystems sad.